packages/by-name/kata-debug-shell/kata-debug-shell.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+# Copyright 2024 Edgeless Systems GmbH
+# SPDX-License-Identifier: BUSL-1.1
+
+set -euo pipefail
+
+if [[ -z ${1:-} || ${1:-} == "--help" ]]; then
+  cat <<EOF >&2
+Usage: $0 <namespace/pod-name>
+
+Utility script to get a debug shell in a Kata Containers sandbox VM.
+
+The script collects the following information from Kubernetes/containerd:
+  - runtimeClass of the pod
+  - sandbox ID of the pod
+With this, it calls the 'kata-runtime' binary installed as part of the runtimeClass,
+using the config file from the same runtimeClass and the sandbox ID to get a shell.
+EOF
+  exit 1
+fi
+
+readonly ctr_namespace="k8s.io"
+
+die() {
+  echo "Error: $1" >&2
+  exit 1
+}
+
+if [[ "$(id -u)" -ne 0 ]]; then
+  die "Please run as root"
+fi
+
+if command -v k3s &>/dev/null; then
+  ctr_cmd="k3s ctr"
+elif command -v ctr &>/dev/null; then
+  ctr_cmd="ctr"
+else
+  die "Neither 'k3s ctr' nor 'ctr' command found. Please install one of them."
+fi
+
+pod_namespace="${1%/*}"
+pod_name="${1##*/}"
+
+if [[ -z $pod_namespace ]] || [[ -z $pod_name ]]; then
+  die "Could not parse namespace or podname from input '$1'."
+fi
```

Comment:
```suggestion
pod_namespace="${1%/*}"
pod_name="${1##*/}"

if [[ "$1" != */*  ]] || [[ -z $pod_namespace ]] || [[ -z $pod_name ]]; then
  die "Could not parse namespace or podname from input '$1'."
fi
```

Assuming that `$1` not containing any `/` should also result in the same error; right now, something like `./kata-debug-shell no-slashes` results in
```
Error from server (NotFound): namespaces "op" not found
Error: Failed to get runtime class for pod 'op' in namespace 'op'.
```

Reply:
good catch :)